### PR TITLE
ticket page added

### PR DESCRIPTION
**Pull request by Imtiyaz-78**
ticket page added

### CODE REVIEW

File: src/Scss/base/_responsive.scss [hunk 1]
```diff
@@ -284,6 +289,37 @@ $largeDesktop: rem(1920);
...
+
+  // Ticket Page
+  .tickets_details {
+    .table_footer {
+      justify-content: center !important;
+      align-items: center !important;
+    }
+  }
+
+  .tickets_search {
```
Comment (TarunRaghav17) [inline src/Scss/base/_responsive.scss:301]: why didn't you used flex mixin for this? and whats the need of important in every property.

File: src/Scss/base/_responsive.scss [hunk 2]
```diff
@@ -284,6 +289,37 @@ $largeDesktop: rem(1920);
...
+    height: auto !important;
+  }
+  .left_items {
+    flex-grow: 1;
+    input {
+      width: 100% !important;
+    }
+  }
+  .table_footer {
+    justify-content: center !important;
```
Comment (TarunRaghav17) [inline src/Scss/base/_responsive.scss:315]: Even here, you should have used flex mixin for this and whats the need of important in every property.

**Review by TarunRaghav17 — CHANGES_REQUESTED** [review]

File: src/Scss/base/_responsive.scss (hunk 2 above)
Comment (Imtiyaz-78) [reply src/Scss/base/_responsive.scss:315]: Inside this style (tickets.component.scss), I have already applied `display: flex` and `justify-content: flex-start`, so I used `!important` to override the existing `display: flex` that was already applied. Therefore, I did not use `flex` here.

File: src/Scss/base/_responsive.scss (hunk 1 above)
Comment (Imtiyaz-78) [reply src/Scss/base/_responsive.scss:301]: Inside a mixin, we can't use `!important` to override an existing property, so I use a normal property and apply `!important` to override the existing property.

**Review by TarunRaghav17 — APPROVED** [review]

**Pull request merged**